frontend/src/pages/Home/HomePage.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState, useEffect } from 'react';
+import { useSelector } from 'react-redux';
+import axios from 'axios';
+import Loading from './components/ProductList/Loading/Loading';
+import { ProductProps } from './components/ProductList/Product/Product';
+import { rootState } from '../../assets/interface/interface';
+import Criteria from './components/Criteria/Criteria';
+import ProductList from './components/ProductList/ProductList';
+import './HomePage.scss';
+
+const HomePage: React.FC = () => {
+  const [products, setProducts] = useState<ProductProps[]>([]);
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState('');
+  const dataUrl = useSelector((state: rootState) => state.filter.url);
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        setLoading(true);
+        const { data } = await axios.get(dataUrl);
+        setProducts(data);
+      } catch (e: any) {
+        setError(e.message);
+      } finally {
+        setLoading(false);
+      }
+    };
+    fetchData();
+  }, [dataUrl]);
+
+  return (
+    <div className="home-page">
+      <div className="shopping">
+        <div className="shopping__container">
+          <div className="shopping__criteria--left">
+            <Criteria type="categories" title="All Categories" criteria={['New Arrivals', 'Featured']} />
```

Comment:
can we declare a const for all criteria:
```
Ex: CATERIA: {
  categories: {
    type: 'categories',
    title: 'abcc',
    .....
  },
...
```
after that we can use it like this `<Criteria {...CATERIA.categories} />`